service/src/main/java/uk/gov/hmcts/reform/fpl/enums/ChildLivingSituation.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.Getter;
+
+import java.util.stream.Stream;
+
+@Getter
+public enum ChildLivingSituation {
+    HOSPITAL_SOON_TO_BE_DISCHARGED("In hospital and soon to be discharged"),
+    REMOVED_BY_POLICE_POWER_ENDS("Removed by Police, powers ending soon"),
+    VOLUNTARILY_SECTION_CARE_ORDER("Voluntarily in section 20 care order"),
+    NOT_SPECIFIED("Not Specified");
+
+    private final String value;
+
+    ChildLivingSituation(String value) {
+        this.value = value;
+    }
```

Comment:
If we are using lombok annotations for the getter then this can be replaced with a `@RequiredArgsConstructor`